### ISSUE TITLE
about/debug crashes on latest console

### ISSUE THREAD

**Issue by meijeru** [red/red#3329]
### Expected behavior
`about/debug` should give expected output
### Actual behavior
Crash
### Steps to reproduce the problem
type ` about/debug` into console
### Red and platform version
```
Red 0.6.3 for Windows built 11-Apr-2018/14:56:31+02:00 commit #9b085fa
W10
```

**Comment by greggirwin** [comment]
Comfirmed, and `about` is very complex now, with some code that could use a cleanup. e.g. the block being `compose`d has no parens, so `compose` is not needed. If we can get it working, so I can see the expected output, I'll get with @x8x and review.

**Comment by x8x** [comment]
https://github.com/red/red/pull/3335 fix the crash by re-adding a previous fix that was removed, see the PR for more info.

**Comment by 9214** [comment]
Still experiencing it on W10 with latest build.

**Comment by greggirwin** [comment]
This one still really bothers me. If we can't get it solved, I'll move to revert to the old `about` code, which was stable.

**Comment by greggirwin** [comment]
I hereby move that we revert to the old `about` code for the 0.6.4 release. What say ye?

**Comment by endo64** [comment]
The problem is not `about` but this issue: https://github.com/red/red/issues/3411

There is this line in `about`: `to-UTC-date git/date` which leads to crash. We should either remove that part or fix the issue. Latter is preferred.

**Comment by 9214** [comment]
Good catch @endo64.

**Comment by greggirwin** [comment]
This seems like it should be easy, and I'm fine removing the date part, but it's been there for months and I don't know if the extra info has proven useful yet.

**Comment by endo64** [comment]
Or we can just remove the `to-UTC-date` and it will show the date with timezone which is fine: `30-Oct-2018/9:30:29+03:00`

This would be the easiest fix.

**Comment by greggirwin** [comment]
I'm good with that as well, though then we need to know where the zone comes from (server wise). Ultimately, we should figure out why that crashes it, but make it a separate ticket.

**Comment by greggirwin** [comment]
This is @x8x's code, so I hope he'll weigh in.

**Comment by greggirwin** [comment]
Thanks @xqlab! Is that a reproducible problem we can put in another ticket?

**Comment by xqlab** [comment]
probably

**Comment by endo64** [comment]
I don't think we need a separate ticket for it, #3411 is already for that issue.

**Comment by xqlab** [comment]
true

**Comment by greggirwin** [comment]
The info here about `return` or setting a time and not an integer should help @qtxie track it down much faster.

**Comment by greggirwin** [comment]
@xqlab did you test your `0:0` fix, it doesn't fix the issue for me.

**Comment by greggirwin** [comment]
Removing the `return` spec doesn't fix it either.

**Comment by greggirwin** [comment]
@xqlab is that in the GUI console, or the CLI console? Only the GUI console has the issue.

**Comment by greggirwin** [comment]
You also need to compile the console, not just paste the code in.

**Comment by greggirwin** [comment]
Which brings up a point about the debug info, which doesn't tell you which console you're in.

**Comment by endo64** [comment]
Interesting thing is if you copy of `to-UTC-date` to a new function:
```
f: func ["Returns the date with UTC zone" 
    date [date!] 
    return: [date!]
][
    date/timezone: 0 
    date
]
```

and also overwrite `to-UTC-date` function:

```
to-UTC-date: func ["Returns the date with UTC zone" 
    date [date!] 
    return: [date!]
][
    date/timezone: 0 
    date]
```

`f` doesn't crash but `to-UTC-date` still crashes.

```
>> f now
== 17-Nov-2018/19:04:33
>> to-UTC-date now
== crash!
```

**Comment by endo64** [comment]
Interesting.. On my second try after overwriting `to-UTC-date` it didn't crash.

**Comment by xqlab** [comment]
> 
> 
> @xqlab is that in the GUI console, or the CLI console? Only the GUI console has the issue.

Of course, this is the gui-console, as the cli console does not crash.   I tested it with an older version of the gui-console. Omitting the return is working. Redefining the to-utc-date even with the original version works if there is at least one return after the redefinition and calling to-utc-date. So not sure if the error is really connected to to-utc-date.

**Comment by greggirwin** [comment]
And the angels sang. Thank you @qtxie !

**Comment by endo64** [comment]
Tested on Windows 10, GUI console didn't crash:
```
>> about/debug
-----------RED & PLATFORM VERSION----------- 
RED: [ branch: "master" tag: #v0.6.3 ahead: 1122 date: 21-Nov-2018/23:28:42 commit: #ad4ed9e003c011b624c3abb64f11f742c27634b8 ]
PLATFORM: [ name: "Windows 8" OS: 'Windows arch: 'x86-64 version: 6.2.0 build: 9200 ]
--------------------------------------------
```
